services/director-v2/src/simcore_service_director_v2/models/schemas/constants.py

```diff
@@ -3,6 +3,10 @@
 DYNAMIC_SIDECAR_SERVICE_PREFIX = "dy-sidecar"
 DYNAMIC_PROXY_SERVICE_PREFIX = "dy-proxy"
 
+# label storing scheduler_data to allow service
+# monitoring recovery after director-v2 reboots
+DYNAMIC_SIDECAR_SCHEDULER_DATA_LABEL = "io.osparc.scheduler-data"
```

Comment:
I do have a question here. Why ```io.osparc.```?
all the other labels are using ```io.simcore``` why this change?

Reply:
confused label, will use `io.simcore`